src/interfaces/api-filing-interface.ts

```diff
@@ -96,12 +96,10 @@ export interface ApiFilingIF {
     continuationOut?: {
       continuationOutDate: IsoDatePacific
       courtOrder?: any
-      details: string
-      foreignJurisdiction: {
-        country: string
-        region?: string
-      }
+      country: string
+      details?: string
       legalName: string
+      region: string // may be null
```

Comment:
This is the filing data we get back from the Legal API. (Unfortunately it doesn't follow the JSON that is used when PUTTING/POSTING a filing.)

Example:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/25f64f3a-f691-4a3a-8f40-635fa0f15d40)